.prettierignore

```diff
@@ -0,0 +1,12 @@
+# Ignore artifacts:
+node_modules
+
+build
+coverage
+dist
+
+optimizeImages
+src/__tests__
+
+# Ignore all HTML files:
+**/*.html
```

Comment:
If we'll need it after Prettier's removal.

Reply:
Removed